.github/workflows/manual-deploy-testnet-l1.yml

```diff
@@ -76,53 +68,52 @@ jobs:
         # DOCKER_BUILDKIT=1 will enable  the new docker build kit that allows us to use build only caches on RUN commands.
         # Tag the same image with 2 tags and push with the -a flag which pushes all images
         run: |
-          DOCKER_BUILDKIT=1 docker build -t ${{env.ETH2NETWORK_BUILD_TAG}} -t ${{env.L1_DOCKER_BUILD_TAG}} -f testnet/eth2network.Dockerfile .
-          docker push ${{env.L1_DOCKER_BUILD_TAG}}
```

Comment:
You probably confirmed it, just checking, the L1_DOCKER_BUILD_TAG isn't used any longer ?

Reply:
Yeah, couldn't find any references to it in the codebase, bit strange though.